models/DataObject/ClassDefinition/Data/ObjectsMetadata.php

```diff
@@ -1057,4 +1072,52 @@ protected function buildUniqueKeyForAppending($item)
 
         return $elementType . $id;
     }
+
+    /**
+     * @param $array1
+     * @param $array2
+     * @return bool
+     */
+    public static function isEqual($array1, $array2)
```

Comment:
@weisswurstkanone if possible, put this method into a trait to share it with MultihrefMetadata